internal/webserver/webserver.go

```diff
@@ -150,7 +150,7 @@ func (n kubeFilter) namespacesHandler(writer http.ResponseWriter, request *http.
 		q.Set("labelSelector", v)
 		log.V(4).Info("labelSelector updated", "selector", v)
 	} else {
-		q.Add("labelSelector", s.String())
+		q.Set("labelSelector", s.String())
```

Comment:
What's the rationale behind the `Set` rather than `Add`?

Reply:
My understanding is:

- `Set` sets the key to value. It replaces any existing values.
  The original query `{"query": "labelSelector"}` becomes `{"query": "labelSelector=capsule.clastix.io%2Ftenant+in+%28gas%2Coil%2Cspectre%29"}`

- `Add` adds the value to key. It appends to any existing values associated with key.
  The original query `{"query": "labelSelector"}` becomes `{"query": "labelSelector=&labelSelector=capsule.clastix.io%2Ftenant+in+%28gas%2Coil%2Cspectre%29"}`

The latter is executed as an OR and hence all the namespaces are returned